include/FreeRTOS.h

```diff
@@ -484,6 +484,12 @@
     #define configUSE_CORE_AFFINITY    0
 #endif /* configUSE_CORE_AFFINITY */
 
+#if ( ( configNUMBER_OF_CORES > 1 ) && ( configUSE_CORE_AFFINITY == 1 ) )
+    #ifndef configTASK_DEFAULT_CORE_AFFINITY
+        #define configTASK_DEFAULT_CORE_AFFINITY tskNO_AFFINITY
+    #endif
+#endif
```

Comment:
How does this help? The check is the same and the assign conditions look the same

Reply:
Can leave the check out. The define is not required if the two options are not set in that way. Defining it always doesn't harm either